ship-it-inc/ship-it
# #167446487 Set up the project

**What does this PR do?**
It set up the project

**Description of Task to be completed?**
- set up server
- set up eslint
- set up sequlize
- set up babel

**How should this be manually tested?**
- Fetch the branch `git fetch ch-project-setup-167446487`
- Check out into the branch `ch-project-setup-167446487`
- run sequelize db:migrate
- start the server with npm start
- The server should be started successfully

**Any background context you want to provide?**
NIL

**What are the relevant pivotal tracker stories?**

[167446487](https://www.pivotaltracker.com/story/show/167446487)

**Screenshots (if appropriate)**

NIL
